### DISCUSSION TITLE
Moving Forward

### DISCUSSION THREAD

**Discussion by ea7kir** [labtroll/KiCad-DesignRules#7] [Q&A]
There are 3 issues and 2 pull requests, in addition to the comments in the KiCad forum.   So I'm asking, if any of these are being considered?  In other words, it would be encouraging to see the author responding by way of accepting and merging or rejecting.

**Comment by Cimos** [comment]
Maybe we need to fork this and someone else picks up the mantel.

I might do this...

**Reply by Cimos** [reply]
FYI: https://github.com/Cimos/KiCad-CustomDesignRules
